website/src/pages/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { Auth, ThemeSupa } from "@supabase/auth-ui-react";
+import { useSession, useSupabaseClient } from "@supabase/auth-helpers-react";
+
+import Head from "next/head";
+
+import { CallToAction } from "../components/CallToAction";
+import { Faq } from "../components/Faq";
+import { Footer } from "../components/Footer";
+import { Header } from "../components/Header";
+import { Hero } from "../components/Hero";
+
+import styles from "../styles/Home.module.css";
+
+export default function Home() {
+  const session = useSession();
+  const supabase = useSupabaseClient();
+
+  const signinWithDiscord = async () => {
+    const { data, error } = await supabase.auth.signInWithOAuth({
```

Comment:
can you also switch this to use the next-auth login methods?

Reply:
Done